src/components/SearchMap.js

```diff
@@ -10,16 +10,16 @@ import {Button} from 'react-native-elements';
 // our components - core
 // our components - additional
 
-const SearchMap = ({mapLoaded, region, updateRegion, sites}) => {
+const SearchMap = ({mapLoaded, lastKnownRegion, updateRegion, sites}) => {
     const {fillScreen, spinnerContainerStyle} = styles;
 
     const newRegionIsAcceptable = (newRegion) => {
         const {longitude, latitude, latitudeDelta, longitudeDelta} = newRegion;
 
-        const notZoomedTooFarOut = latitudeDelta < 60 && longitudeDelta < 60;
-        const isNotSF = (Math.abs(longitude - 122.409) > 1) && (Math.abs(latitude - 37.787) > 1);
+        const zoomedTooFarOut = latitudeDelta > 60 || longitudeDelta > 60;
+        const isSF = (Math.abs(Math.abs(longitude) - 122.409) < 1) && (Math.abs(Math.abs(latitude) - 37.787) < 1);
 
-        return isNotSF && notZoomedTooFarOut;
+        return !isSF && !zoomedTooFarOut;
```

Comment:
Switched these checks to be truthy so we can use the `!` operator, and avoid double-negatives in the future.